src/i18n/locales/de-DE.json

```diff
@@ -276,6 +276,7 @@
   "cmps.designer.lightning.LightningDetails.actions": "Aktionen",
   "cmps.designer.lightning.LightningDetails.waitingNotice": "Warten, bis {{implementation}} online ist",
   "cmps.designer.lightning.LightningDetails.connectError": "Verbindung zu {{implementation}} Knoten nicht möglich",
+  "cmps.designer.lightning.LightningDetails.menuRename": "Umbenennen",
```

Comment:
FYI, you only need to create new keys in the `en-US` file. The other languages are managed by the integration with Crowdin. It will generate the translations and update the other language files in a separate PR after this gets merged into `master`. So you can revert your changes to the non english files.